### PR TITLE
chore(README): add instructions to run unit tests

### PR DESCRIPTION
**Pull request by cironunes**
@IgorMinar can you take a look please?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,6 +62,21 @@ ng build
...
+project once you can use:
+
+```bash
+ng build
+```
+
+With the project built in the `dist/` folder you can just run: `karma start`. 
+Karma will run the tests and keep the browser open waiting to run again.
+
+To watch the files and build them after any change you can use `ng serve`.
```
Comment (IgorMinar) [inline README.md]: I don't think that this is true. We'll need to copy DestCopy broccoli plugin from angular/angular into the pipeline to make this work. am I wrong?

File: README.md [hunk 2]
```diff
@@ -62,6 +62,21 @@ ng build
...
+
+
```
Comment (IgorMinar) [inline README.md]: Maybe leave a note with link to issue to make big test work?

**Comment by IgorMinar** [comment]
The rest looks good

File: README.md (hunk 1 above)
Comment (cironunes) [inline README.md]: This is actually a temporary solution. I'm actually not sure, but might be right.

Comment (IgorMinar) [inline README.md]: I don't think it is. Try it.

File: README.md (hunk 2 above)
Comment (cironunes) [inline README.md]: Added the note.

Comment (IgorMinar) [inline README.md]: Lgtm

File: README.md (hunk 1 above)
Comment (cironunes) [inline README.md]: Created #81.

**Pull request merged**